cmd/azbrowse/main.go

```diff
@@ -341,6 +346,106 @@ func main() {
 	}); err != nil {
 		log.Panicln(err)
 	}
+	if err := g.SetKeybinding("", gocui.KeyCtrlU, gocui.ModNone, func(g *gocui.Gui, v *gocui.View) error {
+
+		item := list.CurrentExpandedItem()
+		if item == nil ||
+			item.SwaggerResourceType == nil ||
+			item.SwaggerResourceType.PutEndpoint == nil {
+			return nil
+		}
+
+		tmpFile, err := ioutil.TempFile(os.TempDir(), "azbrowse-")
+		if err != nil {
+			status.Status(fmt.Sprintf("Cannot create temporary file: %s", err), false)
+			return err
+		}
+
+		err = tmpFile.Close()
+		if err != nil {
+			status.Status(fmt.Sprintf("Cannot close temporary file: %s", err), false)
+			return err
+		}
+
+		newFilename := tmpFile.Name() + ".json"
+		err = os.Rename(tmpFile.Name(), newFilename)
```

Comment:
You can use `*` as a wildcard in `ioutil.TempFile` like https://play.golang.org/p/gSZPfMM8YPV which hopefully removes the need for the rename unless it's used for anther reason